### DISCUSSION TITLE
Any way to disable the sign-up to allow a single org to be created on Self hosted

### DISCUSSION THREAD

**Discussion by ghost** [graphql-hive/console#5289] [Help] [answered]
We would like to prevent users to create new orgs to get a single one with single way of authentication.
Is there a way to create a single tenant of hive? (I know that blocking the routes might be an options but if some got ideas)

**Comment by dotansimha** [comment, marked as answer]
At the moment, this is not supported. I think it can be easily added to the Hive with an env var - contributions are always welcome!
